CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/sidecar-injector#72](https://github.com/cyberark/sidecar-injector/pull/72) <br>
   [cyberark/sidecar-injector#73](https://github.com/cyberark/sidecar-injector/pull/73) <br>
   [cyberark/sidecar-injector#74](https://github.com/cyberark/sidecar-injector/pull/74) <br>
+  [cyberark/sidecar-injector#79](https://github.com/cyberark/sidecar-injector/pull/79) <br>
 
 ### Changed
```

Comment:
should also add the details on what was changed/added

Reply:
the change goes with the above description, specific changes aren't needed for this.